master/getting-started/kubernetes/hardway/istio-integration.md

```diff
@@ -35,9 +35,67 @@ total 5.0M
 ```
 {: .no-select-button}
 
-## Enable application layer policy
+## Enable policy sync API
+
+Application layer policy requires the policy sync API to be enabled on Felix. To do this cluster-wide, modify the `default`
+FelixConfiguration to set the field `policySyncPathPrefix` to `/var/run/nodeagent`.  The following example uses `sed` to modify your
+existing default config before re-applying it.
+
+```bash
+calicoctl get felixconfiguration default --export -o yaml | \
+sed -e '/  policySyncPathPrefix:/d' \
+    -e '$ a\  policySyncPathPrefix: /var/run/nodeagent' > felix-config.yaml
+calicoctl apply -f felix-config.yaml
+```
+
+## Installing Istio
+
+Install Istio 1.1.7, including mutually authenticated TLS for service to service communication.
+
+```bash
+curl -L https://git.io/getLatestIstio | ISTIO_VERSION=1.1.7 sh -
+cd $(ls -d istio-*)
+kubectl apply -f install/kubernetes/helm/istio-init/files/
+kubectl apply -f install/kubernetes/istio-demo-auth.yaml
+```
+
+> **Note**: If an "unable to recognize" error occurs after applying `install/kubernetes/istio-demo-auth.yaml` it is likely a race
+> condition between creating an Istio CRD and then a resource of that type. Re-run the `kubectl apply`.
+{: .alert .alert-info}
+
+## Updating the Istio sidecar injector
+
+The sidecar injector automatically modifies pods as they are created to work
+with Istio. This step modifies the injector configuration to add Dikastes, a
+{{site.prodname}} component, as sidecar containers.
+
+Apply the following ConfigMap to enable injection of Dikastes alongside Envoy.
+
+   ```bash
+   kubectl apply -f {{site.url}}/v3.8/manifests/alp/istio-inject-configmap-1.1.7.yaml
+   ```
+
+## Adding {{site.prodname}} authorization services to the mesh
+
+Apply the following manifest to configure Istio to query {{site.prodname}} for application layer policy authorization decisions
+
+```bash
+kubectl apply -f {{site.url}}/{{page.version}}/manifests/alp/istio-app-layer-policy.yaml
```

Comment:
Should this one also be set to v3.8 instead of page.version?

Reply:
Yeah, good catch.  That manifest hasn't changed in ages, but should also get pinned to shield us from future changes.

Reply:
Fixed. Ready for re-review.